### DISCUSSION TITLE
Help with yggdrasilctl getpeers API in v0.5

### DISCUSSION THREAD

**Discussion by d47081** [yggdrasil-network/yggdrasil-go#1075] [Q&A] [answered]
v0.5 has different CLI response, for example - removed `bytes_send`, `bytes_received`, `coords`, some fields renamed.

Currently, JSON api returns following fields:
```
URI            	State	Dir	              IP Address              	Uptime	 RX  	 TX  	Pr	Last Error
```

The response is intuitively clear, I just need help with `RT`, `XT` and `Pr` understanding to upgrade [YGGstate](https://github.com/YGGverse/YGGstate) libraries.

Or maybe some docs available?
Thanks

**Comment by neilalexander** [comment, marked as answer]
RX ("received") is `bytes_recvd`, TX ("transmitted") is `bytes_sent`, Pr is `priority`.

**Reply by d47081** [reply]
Just found I forgot to provide `-json` flag - it's old names there (just `coors` field changed)
